paparazzi.sh

```diff
@@ -144,8 +144,15 @@ case "$1" in
         ./configure --prefix=/usr
         make -j $N_CORES
         sudo make install
+        # Do some linking
+        sudo rm /usr/lib64/libfreetype.so
```

Comment:
maybe sure to let the user know in the readme what giving sudo permission is here!